docs/07_how-to/03_join-expose.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: 'How to join tables and make the results available'
+sidebar_label: 'Join and expose tables'
+id: ht-join-expose
+keywords: [views, tables, data server, request server]
+tags:
+    - views
+    - tables
+    - data server
+    - request server
+---
+
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+## Joining tables
+Typically, you design the data for your application so that there are neat places to store all the different types of data. You could have separate tables for INSTRUMENTS, CURRENCIES and COUNTERPARTIES, for example. And then you might have more tables for ORDERS, TRADES and CLIENT_DETAILS.
+
+However, when you want to display data and provide it for reports, you want a view that takes data from more than one table.
```

Comment:
"from the joined table" instead of more than one table?

Reply:
no: specifying the use case at this point, not the implementation